packages/ui/Tooltip.tsx

```diff
@@ -12,6 +15,7 @@ export function Tooltip({
   children: React.ReactNode;
   content: React.ReactNode;
   open?: boolean;
+  side?: "top" | "right" | "bottom" | "left" | undefined;
```

Comment:
```suggestion
  position?: "top" | "right" | "bottom" | "left" | undefined;
```

Reply:
I also thought this, then Googled and in Bootstrap for example it's called `placement` - position there implies the orientation. Naming in Tooltip world is quite messy.

Reply:
Oh well. Placement still sounds better than side to me though.

Reply:
`side` is actually the native prop that comes from radix, which is why I used it

<img width="265" alt="image" src="https://user-images.githubusercontent.com/8019099/178968336-0f983b32-29a5-4cdb-9761-fb090a3958bb.png">